types/tokenizr/index.d.ts

```diff
@@ -0,0 +1,264 @@
+// Type definitions for tokenizr 1.5
+// Project: https://github.com/rse/tokenizr
+// Definitions by: Nicholas Sorokin <https://github.com/aNickzz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export {};
```

Comment:
I think this line can be removed

Reply:
I added this as a recommendation from the linter as `Action` and `RuleAction` are not explicitly (or implicitly) exported.

>All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting.

Those types are not part of tokenizrs exports, I just didn't want to copy/paste the definitions in the places its used.